site/content/en/guides/consumer/set/_index.md

```diff
@@ -97,7 +97,9 @@ Print the list of setters included in the package.
     NAME      VALUE       SET BY             DESCRIPTION        COUNT  
   http-port   80      package-default   helloworld port         3      
   image-tag   0.1.0   package-default   hello-world image tag   1      
-  replicas    5       package-default   helloworld replicas     1     
+  replicas    5       package-default   helloworld replicas     1      
+  SUBSTITUTION                      PATTERN                       REFERENCES   
+  image-tag      gcr.io/kpt-dev/helloworld-gke:IMAGE_TAG_SETTER   [image-tag]
```

Comment:
Per new changes, a setter and substitution should not have same name. We should rename the substitution in the hello world package, update the version tag and point it here.

Reply:
Fixed